pkg/scheduler/framework/arguments.go

```diff
@@ -0,0 +1,28 @@
+package framework
```

Comment:
add license there.

Reply:
copy license from other file.

And I think the license may be outdated now? 
```
Copyright 2018 The Kubernetes Authors
```

Reply:
Could use 2019 for new added files

Reply:
New added files (arguments.go arguments_test.go) is now in license of 2019.